yarl/quoting.py

```diff
@@ -1,7 +1,16 @@
+import os
 import re
+import sys
 from string import ascii_letters, ascii_lowercase, digits
 from typing import Optional, TYPE_CHECKING, cast
 
+
+NO_EXTENSIONS = bool(os.environ.get("YARL_NO_EXTENSIONS"))  # type: bool
```

Comment:
How about having `EXTENSIONS` instead of `NO_EXTENSIONS`? Double negation like `not NO_EXTENSIONS` looks confusing.

Reply:
`NO_EXTENSIONS` corresponds to env var, `EXTENSIONS` is for *not env var* which is an extra step in the logic understanding.